cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -638,17 +693,22 @@ protected PersistentVolumeClaim createPersistentVolumeClaim(int podNumber, Strin
 
         String storageClass = storage.getStorageClass();
         if (storage.getOverrides() != null) {
-            storageClass = storage.getOverrides().stream().filter(broker -> broker != null && broker.getBroker() != null && broker.getBroker() == podNumber && broker.getStorageClass() != null)
+            storageClass = storage.getOverrides().stream()
+                    .filter(broker -> broker != null &&
+                            broker.getBroker() != null &&
+                            broker.getBroker() == ordinalId &&
+                            broker.getStorageClass() != null)
```

Comment:
I do not think there is a real rule for this or anything like that, so feel free to leave it like this ... But I usually prefer to have the `&&` at the next line - it seems to me more readable.

```suggestion
                    .filter(broker -> broker != null 
                            && broker.getBroker() != null 
                            && broker.getBroker() == ordinalId 
                            && broker.getStorageClass() != null)
```

Reply:
Great suggestion!